NewsApp/NewsApp/Model/Services/NewsService.swift

```diff
@@ -12,12 +12,17 @@ protocol NewsService {
     var newsArticles: [Article] { get }
     var parametersRequest: ParametersRequest { get }
     func fetchArticles(completion: @escaping () -> Void)
+    func loadMore()
+    var allowToLoadMore: Bool { get set }
```

Comment:
Je pense que allowToLoadMore doit être interne à NewsService et pas accessible

Reply:
fix in "fix PR"